Makefile

```diff
@@ -74,14 +74,14 @@ $(BINARIES): bin/%:
 mod-tidy:
 	$(Q)for mod in $$(find . -name go.mod); do \
 	    echo "Tidying $$mod..."; ( \
-	        cd $$(dirname $$mod) && go mod tidy \
+	        cd $$(dirname $$mod) && go mod tidy -go=$$(grep -E "^go\s+[1-9]\.[0-9]+$$" go.mod | sed 's/^go //') \
```

Comment:
I'm still not sure I understand the intent of the change.
According to the `go mod tidy` help output
```
The -go flag causes tidy to update the 'go' directive in the go.mod
file to the given version
```

However, updating the directive to the version from the same file does nothing, doesn't it?